<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenimage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowhist">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>531</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnhanceimage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enhance image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectWebcam">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopWebcam">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvertCam">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Blurring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSmooth">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoothing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSharpen">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sharpening</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtBlur">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtRotate">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScale">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFlipTB">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FlipTB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWarpRows">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>WarpRows</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCrop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFlipLR">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FlipLR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWarpCols">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>WarpCols</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtScaleCol">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtScaleRow">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotateCam">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>130</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Rotate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtRotateCam">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>130</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCropX1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCropX2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCropY1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCropY2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveJPEG">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save to JPEG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSavePNG">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save to PNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveBMP">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save to BMP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
